### PR TITLE
Fix partial i18n of line

### PR DESCRIPTION
**Pull request by slichtzzz**
Fix partial i18n of line

### CODE REVIEW

File: gui/wxpython/gui_core/forms.py [hunk 1]
```diff
@@ -580,7 +580,7 @@ def __init__(
         guisizer.Add(self.notebookpanel, proportion=1, flag=wx.EXPAND)
 
         # status bar
-        status_text = _("Enter parameters for '") + self.task.name + "'"
+        status_text = _("Enter parameters for '") + self.task.name + _("'")
```
Comment (wenzeslaus) [inline gui/wxpython/gui_core/forms.py:583]: This needs to be done like the other change you have. Use `_("...{name}...").format(name=...)`. Then, the translator can translate the whole message at once (each `_(...)` is a separate message to translate.

Comment (slichtzzz) [reply gui/wxpython/gui_core/forms.py:583]: Ok. Thank you!

**Review by wenzeslaus — COMMENTED** [review]
The original self.ptitle code is sub-optimal and relies on the non-translated messages. The correct implementation would be

```python
pairs = []
for ...:
    pairs.append(_("...vs..."))
self.ptitle =  _("Bivariate Scatterplot of {list_of_pairs}".format(list_of_pairs=_(", ").join(pairs))
```

I'm not sure how much to change in this PR or if my suggestion is the best for translations. Would you consider reverting your piece here and pursuing this in a separate PR?

**Comment by slichtzzz** [comment]
Hi! I'm sorry. I'm trying to put the commit in a new pull request, but despite all my efforts all the stuff is going back to the same old pull request. Apparently, I'm doing something wrong, but don't see how to fix it…

**Comment by wenzeslaus** [comment]
> ...put the commit in a new pull request, but despite all my efforts all the stuff is going back to the same old pull request...

On your local machine, you interact with branches, the commits go to branches, then you open one PR per branch.

Your branch for this PR is main which is not great, but not a deal breaker. The best next step depends on whether you are using a local Git repo or doing things in the GitHub web interface only. Trying both ways may give you some insights. On web, you are trying to modify your fork, try to modify this repo and it should lead you the right way.

In any case, check the CONTRIBUTING.md file and some basic tutorials. I think it is a confusing process the first time around, but I don't have any remedy for that.

File: gui/wxpython/rlisetup/wizard.py [hunk 2]
```diff
@@ -1230,11 +1230,8 @@ def afterRegionDrawn(self, marea):
             self.parent.wizard.ShowPage(self.parent.summarypage)
         else:
             self.title.SetLabel(
-                _(
-                    "Draw sample region "
-                    + str(self.regioncount)
-                    + " of "
-                    + str(numregions)
+                _("Draw sample region {count} of {num_regions}").format(
+                    count=self.regioncount, num_regions=numregions
```
Comment (echoix) [inline gui/wxpython/rlisetup/wizard.py:1234]: Does python need more than this to format numbers correctly, or the default string representation is enough?

File: gui/wxpython/wxplot/scatter.py [hunk 3]
```diff
@@ -134,24 +134,25 @@ def OnSelectRaster(self, event):
...
             # update title
-            self.ptitle += "%s vs %s, " % (
-                rpair[0].split("@")[0],
-                rpair[1].split("@")[0],
+            pairs.append(
+                _("{rpair0} vs {rpair1}").format(
+                    rpair0=rpair[0].split("@")[0], rpair1=rpair[1].split("@")[0]
+                )
+            )
+            self.ptitle = _(
```
Comment (HuidaeCho) [inline gui/wxpython/wxplot/scatter.py:152]: This `join()` should ideally be done outside the `for` loop. No need to keep overwriting it. Can replace `self.ptitle = self.ptitle.strip(", ")` below, which is not needed now.

**Comment by echoix** [comment]
This will probably be closed by #4052 once merged. That other PR adresses all of these issues in Python code.

**Comment by echoix** [comment]
Fixed with merged #4052

**Pull request closed without merging**